troposphere/autoscaling.py

```diff
@@ -181,6 +189,7 @@ class AutoScalingGroup(AWSObject):
         'LoadBalancerNames': (list, False),
         'MaxInstanceLifetime': (integer, False),
         'MaxSize': (integer, True),
+        'MetadataOptions': (MetadataOptions, False),
```

Comment:
[`AWS::AutoScaling::AutoScalingGroup`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html) doesn't have this property
```suggestion
```

Reply:
fixed